src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -816,6 +821,46 @@ private DeleteOrganisationResponse deleteOrganisationEntity(Organisation organis
         return deleteOrganisationResponse;
     }
 
+    @Override
+    @Transactional
+    public DeleteUserResponse deleteUserForOrganisation(List<String> emails) {
+        var deleteOrganisationResponse = new DeleteOrganisationResponse();
+        if (emails.isEmpty()) {
```

Comment:
minor: checking for emails could be first, even before creation of response object